src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
Before requesting the review, please make sure that you went through the common mistake checklist: https://github.com/mate-academy/jv-lottery/blob/master/checklist.md
This object should be a field.